index.html

```diff
@@ -139,14 +138,12 @@ <h1>Tonic</h1>
                  </div>
             </div>  
         </div>
-        <div class="portfolio">
-            <div class="snapshot4">
-                <!-- <img src="./images/portfolio3.png" alt="fourth portfolio"> -->
-            </div>
+        <div class="portfolio reverse">
+            <div class="snapshot4"></div>
             <div class="portfolio-content">
                 <div class="title-heading2">
-                    <h1>Multi-Post<br>
-                        Stories</h1>
+                    <h3>Multi-Post
```

Comment:
kindly add a margin-top of 20px to give space between the image and h3 element 😃